src/components/ServerVersion.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from "react";
+import { Icon, Tooltip } from "@canonical/react-components";
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import { fetchSettings } from "api/server";
+
+const Navigation: FC = () => {
+  const { data: settings } = useQuery({
+    queryKey: [queryKeys.settings],
+    queryFn: fetchSettings,
+  });
+
+  const serverVersion = settings?.environment?.server_version;
+  const majorVersion = serverVersion?.includes(".")
+    ? serverVersion.split(".")[0]
+    : undefined;
+  const isOldVersion = majorVersion ? parseInt(majorVersion) < 5 : false;
+  const versionLabel = serverVersion ? `Server ${serverVersion}` : undefined;
+
+  return serverVersion ? (
+    <>
+      <hr className="p-side-navigation__list is-dark navigation-hr" />
+      <li className="p-side-navigation__link server-version">
+        {isOldVersion ? (
+          <Tooltip
+            message="You are using an outdated version.
+Update your LXD server to benefit from the latest features."
+            tooltipClassName="version-warning"
+            zIndex={1000}
+          >
+            <Icon name="warning" className="p-side-navigation__icon" />
```

Comment:
I'm not sure I understood correctly the previous comment from @al-bakalova, but I think only the Icon should be wrapped inside the Tooltip.
Or, to say that in a less technical way, you'd get to see the tooltip only by hovering over the icon, instead that both the icon and the label, no? What is the expected behaviour here @al-bakalova?

Reply:
I think it is better to extend the tooltip to the text and icon hover. I don't see a reason why we would want to not show the tooltip on the "Server 5.13" text hover.

Reply:
Honestly, I don't think people will naturally hover on the label. I'd expect them to hover on the alarming icon though.
I think just an icon hover is fine, but the whole label might be useful too. As long as there is no tooltip when the user is running an okay version.

Reply:
Just looking at the other tooltips that we have in the app, almost all of them are wrapping only an icon. So that seemed to me the right behaviour for this case as well.

Reply:
Yeah, let's keep it consistent, if that's the logic behind. That's a good point

Reply:
It is actually broken, when applying the tooltip only to the icon the position gets calculated wrong. Lets keep it simple where it is - that works :)

![Screenshot from 2023-04-25 15-03-07](https://user-images.githubusercontent.com/1155472/234284957-f05c6f29-cc48-4109-8315-ecd5e011cbdd.png)

Reply:
> It is actually broken, when applying the tooltip only to the icon the position gets calculated wrong.

Fix suggested [here](https://github.com/canonical/lxd-ui/pull/308#discussion_r1176502399)